File-Formats/Presentation/Working-with-find-and-replace.md

```diff
@@ -0,0 +1,932 @@
+---
+title: Presentation Working with Find and Replace | Syncfusion
+description: This section illustrates how to find a particular text and replace it with another text or part of the document
+platform: file-formats
+control: Presentation
+documentation: UG
+keywords: 
+---
+# Working with Find and Replace
+
+.NET PowerPoint library allows you to search a particular text you like to change and replace it with another text in the PowerPoint presentation.
```

Comment:
.NET PowerPoint library allows you to search for a particular text that you want to change and replace it with another text in the PowerPoint presentation.

Reply:
Modified as you requested.